samples/cellular/modem_shell/src/location/location_cmd_utils.c

```diff
@@ -64,7 +64,7 @@ static int location_cmd_utils_generate_nmea(const struct location_data *location
 
 int location_cmd_utils_gnss_loc_to_cloud_payload_json_encode(
 	enum nrf_cloud_gnss_type format, const struct location_data *location_data,
-	char **json_str_out)
+	int64_t ts_ms, char **json_str_out)
```

Comment:
Good catch! Seems this has been a bug in the original development version. Probably unnoticed because we get some time into the cloud map anyway based on when the data is sent.